src/storm/storage/jani/TemplateEdgeDestination.cpp

```diff
@@ -15,8 +15,9 @@ TemplateEdgeDestination::TemplateEdgeDestination(std::vector<Assignment> const&
     // Intentionally left empty.
 }
 
-void TemplateEdgeDestination::substitute(std::map<storm::expressions::Variable, storm::expressions::Expression> const& substitution) {
-    assignments.substitute(substitution);
+void TemplateEdgeDestination::substitute(std::map<storm::expressions::Variable, storm::expressions::Expression> const& substitution,
+                                         bool const& substituteTranscendentalNumbers) {
```

Comment:
bool by const& triggers a warning on my system

Reply:
I can remove the reference. On my system quite some warning are triggered, so it is hard to notice new ones.
Are there more warnings that were added by this PR?

Reply:
I had a look in the repository and `bool const&` seems to be a common pattern for function definitions in STORM.

Of course I can switch the ones I added to be `bool const`, by dropping the reference, but maybe it would make sense to make that consistent throughout the whole repository. The same could be done for all other basic types (int, float, double, char)

Reply:
I changed all declarations of `bool const& substituteTranscendentalNumbers` to not pass the value by reference anymore here: https://github.com/moves-rwth/storm/pull/556/commits/6f016ca1cb72ee0eb2a64cfc97c43f2f33150df0
On my machine, this seems not to throw warnings anymore.